core/src/executor/select/mod.rs

```diff
@@ -298,20 +298,10 @@ pub async fn select_with_labels<'a>(
             Ok((AggregateContext { aggregated, next }, row))
         }
     });
-
-    let rows = sort
-        .apply(rows, labels.clone(), get_alias(relation)?)
-        .await?;
-
-    // let rows = sort
-    //     .apply(rows)
-    //     .await?
-    //     .and_then(move |(aggregated, context)| {
-    //         let blend = Rc::clone(&blend);
-
-    //         async move { blend.apply(aggregated, context).await }
-    //     });
+    let labels = Rc::new(labels);
+    let rows = sort.apply(rows, labels, get_alias(relation)?).await?;
```

Comment:
Is this compiled?
```rust
    let rows = sort.apply(rows, Rc::clone(&labels), get_alias(relation)?).await?;
```
`Rc::clone` looks needed

Reply:
Ahh! I wasn't using Rc at all..
It works now! Thank you so much 🙇🏼